### ISSUE TITLE
Stoploss with wrong qty after a partial takeprofit

### ISSUE THREAD

**Issue by dpanelli** [jesse-ai/jesse#510]
🐞 Bug Report: Stoploss Order Reflects Incorrect Quantity After Partial Take-Profit

Summary:
After executing a partial take-profit order, the corresponding stop-loss order displays an incorrect quantity that does not match the remaining position.

Steps to Reproduce:
	1.	Open a long position with a total quantity of 0.17 at market price (e.g., at 66223.8).
	2.	Set a limit take-profit order for a partial quantity (e.g., 0.09 at 67548.28).
	3.	Set a stop-loss order for the remaining position.
	4.	Allow the take-profit order to partially execute.

Expected Result:
After the partial take-profit is filled, the stop-loss quantity should reflect the remaining position (0.08 in this case).

Actual Result:
The stop-loss quantity remains -0.17, which is the original full position size, even though 0.09 has already been sold.

Screenshot Reference:
	•	Initial BUY: 0.17 @ 66223.8 (MARKET)
	•	Partial TAKE-PROFIT: -0.09 @ 67548.28 (LIMIT SELL)
	•	STOP-LOSS: -0.17 @ 59601.42 (STOP SELL) ❌ Incorrect — should be -0.08

**To Reproduce**
Orders set up code.

       def go_long(self):
          entry_price = self.price
          qty = utils.size_to_qty(self.balance, entry_price)
  
          stop_loss_price = entry_price * 0.9
          take_profit_1_price = entry_price * 1.02
          take_profit_2_price = entry_price * 1.1
  
          self.buy = qty, entry_price
          self.stop_loss = qty, stop_loss_price
          self.take_profit = [
              (qty * 0.5, take_profit_1_price),  
              (qty * 0.5, take_profit_2_price),  
        ]

<img width="371" alt="Image" src="https://github.com/user-attachments/assets/8943ada1-3338-4d66-919a-4683ce10d975" />
